src/SDKs/HealthcareApis/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
1. All the files should be put in **sdk** instead of **src/SDKs** because of structure refactoring
2. Use lower case **heaalthcareapis** for folder name of this service
3. The folder name and project names should start with **Microsoft.Azure.**